frontend/packages/integration-tests-cypress/tests/crud/customresourcedefinition.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+
+import { checkErrors, testName } from '../../support';
+import { nav } from '../../views/nav';
+import { listPage } from '../../views/list-page';
+import * as yamlEditor from '../../views/yaml-editor';
+import { errorMessage } from '../../views/form';
+
+describe('CustomResourceDefinitions', () => {
+  const plural = `crd${testName}`;
+  const group = 'test.example.com';
+  const name = `${plural}.${group}`;
+  const testLabel = 'automatedTestName';
+  const crd = {
```

Comment:
Not new, but would be good to add the type

```suggestion
  const crd: CustomResourceDefinitionKind = {
```